src/app/components/add-card/add-card.component.css

```diff
@@ -11,3 +12,7 @@
 .fa-trash-alt {
   color: rgba(109, 107, 107, 0.835);
 }
+.text.completed {
+  text-decoration: line-through;
+  color: green;
+}
```

Comment:
Add empty new line EOF. Please config `eslint` to do that automatically for you.

Reply:
I tried to configure ESLint for css-modules, but it doesn't seem to be working. After researching, I found suggestions to use stylelint instead. It's possible that there might be a misunderstanding or mistake on my part. please let me know

Reply:
You are right. I mixed with eslint with stylelint. :)